### DISCUSSION TITLE
Frigate unable to read cameras using ipv6 addresses

### DISCUSSION THREAD

**Discussion by tootai** [blakeblackshear/frigate#14096] [General]
Hi all,

my setup today is following, using AgentDVR running in a W10 VM. Cameras are on another location connected with wireguard VPN mounted on VM host. All setup is using ipv6 except cameras on the other end for which I use 6tunnel . Everything is working fine since ages.

Now I want to get rid of this VM and installed on the host frigate/docker. I read documentation on how to enable frigate listening on ipv6 -which is working- but the rtsp stream to cameras still fail with

frigate         | 2024-10-01 19:02:33.411217057  19:02:33.411 WRN [rtsp] error="streams: dial tcp [fd02:d:b:e::7102]:65008: connect: network is unreachable" stream=Camera3

docker0 has an ipv6 ip from the VPN range fd02:d:b:e::/64.

What could be wrong with my configuration or frigate/docker isn't able to do ipv6?

Thanks for any hint

**Comment by NickM-27** [comment]
are you sure that you enabled ipv6? you might also want to make sure that ffmpeg works with ipv6, might need to use a newer version of ffmpeg

**Comment by tootai** [comment]
If I can access my docker using ipv6 I hope it does mean that it's enabled ;) For ffmpeg, how to know?
